src/config/links.ts

```diff
@@ -2,9 +2,12 @@ import { translateRaw } from '@translations';
 import { CRYPTOSCAMDB } from './data';
 import { TURL } from '@types';
 
+import SocialTelegramIcon from '../assets/images/social-icons/social-telegram-white.svg';
+
 interface Link {
   link: string;
   text: string;
+  icon: null | string;
```

Comment:
Would prefer `icon?: string`.

Then it can be `undefined` and thus not necessary to declare `null` every time below.

Reply:
I just pushed up changes